solutions/dotnet_azure_sdk/Makefile

```diff
@@ -10,6 +10,12 @@ DIRS += KeyVault_Certificates
 REDEFINE_TESTS=1
 include $(TOP)/rules.mak
 
+ifdef RUN_AZURE_TESTS
```

Comment:
Under what circumstances we want `RUNTEST` to be empty?

Reply:
When we are running any of the tests independently, in that case runtest would be empty